util/fibers/proactor_base.cc

```diff
@@ -213,9 +213,16 @@ void ProactorBase::Migrate(ProactorBase* dest) {
   me->scheduler()->SuspendAndExecuteOnDispatcher([me, dest] {
     me->DetachScheduler();
 
-    dest->DispatchBrief([me] {
-      me->AttachScheduler();
-    });
+    auto cb = [me] {
```

Comment:
This can explain the crash in the migrate:
```
scheduler.cc:426] Check failed: dispatch_cntx_->list_hook.is_linked()
```
if a dispatcher was preempted here or some other place, the preemption would succeed in release mode due to DCHECK, and 
the scheduler would switch to another fiber with dispatch_cntx_ not being ready.
if that fiber would decide to migrate we would get this check-fail.